projects/astosswap/api.js

```diff
@@ -0,0 +1,37 @@
+const ADDRESSES = require('../helper/coreAssets.json')
+
+const { call, multiCall, parseAddress } = require('../helper/chain/starknet')
+const abi = require('./abi')
+const { transformDexBalances } = require('../helper/portedTokens')
+const { getParamCalls } = require('../helper/utils')
+
+async function tvl() {
+  const factory = '0x07df3bce30857e8f9c08bcd9d9668df34166e94dd968db6e2920b870c4410e34'
```

Comment:
this is stakex factory address

Reply:
Yes 
we are StarkEx AMM team 
our previous project website and twitter is suspended
we just deploy product on this new domain
pls check

Reply:
I see that the discord is still active, can you post the new domain/rebranding info there from admin account as proof?

Reply:
Sure We post it within 5 minutes

Reply:
Just post the announcement
![image](https://github.com/DefiLlama/DefiLlama-Adapters/assets/149654504/a095faea-e28b-4771-89ac-38cf815bcf81)
pls check there

Reply:
We choose AstroSwap AMM